server/services/teamsSummaryCountService.ts

```diff
@@ -216,11 +216,10 @@ export default class TeamsSummaryCountService {
   /**
    * Helper: Function to filter by specified env, or default to prod if none provided
    */
-  async filterTrivyByEnv(env: string): Promise<TrivyScanType[]> {
-    const allTrivyScans = await this.serviceCatalogueService.getTrivyScans()
-
-    // Guard against scans that may not include an environments array
-    return allTrivyScans.filter(trivy => (trivy.environments ?? []).includes(env))
+  async filterTrivyByEnv(env: string[]): Promise<TrivyScanType[]> {
```

Comment:
Sorry being picky 😁 

Could rename this `environment` since it's now plural (can be)

Reply:
The variable as well, currently it's `env` 👍